manager_basic.go

```diff
@@ -173,6 +173,8 @@ func (m *basicProcessManager) Close(ctx context.Context) error {
 	termCtx, cancel := context.WithTimeout(ctx, 5*time.Second)
 	defer cancel()
 
+	defer m.Clear(ctx)
```

Comment:
I worry that we don't always want to clear... Some idea of historic processes would probably be useful.

Reply:
I removed the clear from basic manager.